app/controllers/admin/recent_activity_collections_controller.rb

```diff
@@ -13,11 +13,11 @@ def index
     private
 
     def days_limit
-      params.fetch(:admin_recent_activity, {}).fetch(:days_limit, 7).to_i
+      recent_activity_params.fetch(:days_limit, Admin::RecentActivityForm.new.days_limit).to_i
```

Comment:
Can you make it a constant on the form and reference it here? This will make it clearer than instantiating a form to get the value.